src/common/serverinfo.ts

```diff
@@ -60,16 +60,19 @@ function getServerInfo(connection: Connection): Promise<GetServerInfoResponse> {
   })
 }
 
+// TODO: This was originally annotated to return a number, but actually
+// returned a toString'ed number. Should this actually be returning a number?
```

Comment:
I believe this should be a string. We generally use strings for all currency amounts.

Reply:
sounds good.

Looking at this more closely now, should this use `BigNumber` internally? `Number` could cause problems with `baseFeeXRP` being a float, right?

Reply:
Yes, it would be correct to use `BigNumber` here internally.